packages/renderer/src/crf.ts

```diff
@@ -71,17 +71,28 @@ export const validateQualitySettings = ({
 	codec,
 	crf,
 	videoBitrate,
+	maxRate,
+	bufSize
 }: {
 	crf: unknown;
 	codec: Codec;
 	videoBitrate: string | null | undefined;
+	maxRate: string | null | undefined;
+	bufSize: string | null | undefined;
 }): string[] => {
 	if (crf && videoBitrate) {
 		throw new Error(
 			'"crf" and "videoBitrate" can not both be set. Choose one of either.',
 		);
 	}
 
+	const bufSizeArray = bufSize ? ['-bufsize', bufSize] : [];
```

Comment:
I think bufSize and maxRate are compatible with crf or videobitrate, so I am just spreading in both of them

Reply:
Maybe I missed an error to handle here ?